src/openms/include/OpenMS/CHEMISTRY/SvmTheoreticalSpectrumGeneratorSet.h

```diff
@@ -61,28 +61,13 @@ namespace OpenMS
   class OPENMS_DLLAPI SvmTheoreticalSpectrumGeneratorSet
   {
 public:
-
-    /** @name Constructors and Destructors
-     */
-    //@{
-    /// Default constructor
-    SvmTheoreticalSpectrumGeneratorSet();
-
-    /// Copy constructor
-    SvmTheoreticalSpectrumGeneratorSet(const SvmTheoreticalSpectrumGeneratorSet & source);
-
-    /// Destructor
-    virtual ~SvmTheoreticalSpectrumGeneratorSet();
-    //@}
-
-    /// Assignment operator
-    SvmTheoreticalSpectrumGeneratorSet & operator=(const SvmTheoreticalSpectrumGeneratorSet & tsg);
+    // Rule of Zero in effect for this class.
 
     /// Generate the MS/MS according to the model for the given precursor_charge
     void simulate(PeakSpectrum & spectrum, const AASequence & peptide, boost::random::mt19937_64& rng, Size precursor_charge);
 
     ///Load a trained Svm and Prob. models
-    void load(String);
+    void load(String filename);
```

Comment:
why not `const String& filename` ?

Reply:
If no move semantics are guaranteed, `const String& filename` should be preferred.